src/endpoints/base_endpoint_registry.cpp

```diff
@@ -18,6 +18,36 @@ namespace ccf
     context(context_)
   {}
 
+  ApiResult BaseEndpointRegistry::get_view_history_v1(
+    std::vector<ccf::TxID>& history, ccf::View since)
+  {
+    try
+    {
+      if (consensus != nullptr)
+      {
+        if (since < 1)
+        {
+          // views start at 1
```

Comment:
Actually, CCF views start at 2, for historical reasons. But we might fix that one day.

Reply:
Seemingly so from the user perspective but I think the internal consensus code differs: https://github.com/jeffa5/CCF/blob/13f39e69ce18705fc3b4216b64f42de5ad0bbf31/src/consensus/aft/impl/state.h#L20

I've gone with starting from term 1 (which in practice works out nicely with other endpoints too)